app/src/main/java/com/duckduckgo/app/survey/api/SurveyService.kt

```diff
@@ -27,6 +27,6 @@ interface SurveyService {
     @GET("https://staticcdn.duckduckgo.com/survey/v2/survey-mobile.json")
     fun survey(): Call<SurveyGroup?>
 
-    @GET("https://staticcdn.duckduckgo.com/survey/apptp/retention-survey-mobile.json")
-    fun surveyAppTp(): Call<SurveyGroup?>
+    @GET("https://staticcdn.duckduckgo.com/survey/netp/waitlist-survey-mobile.json")
+    fun surveyNetPWaitlistBeta(): Call<SurveyGroup?>
```

Comment:
Will the appTP survey no longer be needed? Or is the logic already included in the `survey()` call to handle it?

Reply:
yeah, AppTP survey no longer used